src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -20,10 +20,14 @@ package object rlp {
 
   def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
 
+  def encodeToRlp[T](input: T)(implicit enc: RLPEncoder[T]): RLPEncodeable = enc.encode(input)
```

Comment:
Adding this to implicits before Seq one

 `implicit def byteStringFromEncodeable(b: ByteString)(implicit enc: RLPEncoder[ByteString]): RLPEncodeable = enc.encode(b)` 

Allowed me to do 

`RLPList(loggerAddress, logTopics: RLPList, data)`